File: RecyclerView/Java/app/src/main/java/com/example/javarecyclerexample/MainActivity.java
package com.example.javarecyclerexample;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {

    RecyclerView animeRecycle;
    List<Anime> animeList;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        animeList = new ArrayList<>();

        animeList.add(new Anime("Clannad", "Okazaki Tomoya"));
        animeList.add(new Anime("Re: Zero kara Hajimeru Isekai Seikatsu", "Natsuki Subaru"));
        animeList.add(new Anime("Boku no Hero Academia", "Midoriya Izuku"));

        animeRecycle = (RecyclerView)findViewById(R.id.recycler);
        animeRecycle.setLayoutManager(new LinearLayoutManager(this));
        animeRecycle.setAdapter(new AnimeAdapter(animeList));

    }
}

File: RecyclerView/Kotlin/KotlinRecyclerExample/app/src/main/java/com/example/kotlinrecyclerexample/MainActivity.kt
package com.example.kotlinrecyclerexample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var animeList = mutableListOf<Anime>()

        animeList.add(Anime("Clannad", "Okazaki Tomoya"))
        animeList.add(Anime("Re: Zero kara Hajimeru Isekai Seikatsu", "Natsuki Subaru"))
        animeList.add(Anime("Boku no Hero Academia", "Midoriya Izuku"))

        val animeAdapter = AnimeAdapter(animeList)

        recycler.layoutManager = LinearLayoutManager(this)
        recycler.adapter = animeAdapter
    }
}
